{"name": "test_unsuccessful_login[chrome-fttt-ft]", "status": "passed", "attachments": [{"name": "log", "source": "1d0e5910-c080-498b-a8b8-c53dc0c6ace5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "'fttt'"}, {"name": "password", "value": "'ft'"}], "start": 1622283877643, "stop": 1622283878101, "uuid": "b5485681-1bf9-404f-8b9d-b57a782a397e", "historyId": "bce06524281788a71e00192f5e087f3f", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "severity", "value": "normal"}, {"name": "story", "value": "Cloud Login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "4672-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}